README.md

```diff
@@ -1,3 +1,18 @@
 # lyant - Learn You A Next Thing
 
 Bored? Want to learn something?
+
+Added github login, to run the project just open `app->controller->app.go`
```

Comment:
yeah I am working on finding the solution for loading defaults from config file, and also looking forward for your setup.sh script. where we would enter these details.